net_util/src/tap.rs

```diff
@@ -129,6 +129,8 @@ fn ipv6_mask_to_prefix(mask: Ipv6Addr) -> Result<u8> {
 }
 
 impl Tap {
+    /// # Safety
+    /// The `ioctl()` arguments must be correct.
```

Comment:
Let's be more specific: `The caller should ensure to pass a valid file descriptor`.

Reply:
Sure, makes sense to be more explicit! I hope with the updated PR, we are good to go.